src/selection_edge.c

```diff
@@ -93,66 +86,63 @@ void selection_edge_create(void)
     attr.override_redirect = True;
 
     pe->classHint = XAllocClassHint();
-    pe->classHint->res_name  = "scrot";
+    pe->classHint->res_name = "scrot";
     pe->classHint->res_class = "scrot";
 
     pe->wndDraw = XCreateWindow(disp, root, 0, 0, WidthOfScreen(scr), HeightOfScreen(scr), 0,
-          CopyFromParent, InputOutput, CopyFromParent, CWOverrideRedirect | CWBackPixel, &attr);
+        CopyFromParent, InputOutput, CopyFromParent, CWOverrideRedirect | CWBackPixel, &attr);
 
-    unsigned long opacity = opt.line_opacity * ((unsigned)-1 / 100);
```

Comment:
I'm going to reverse this, must be `(unsigned)-1`

Reply:
Yes. Maybe the styling guide should state that we don't allow explicit casts of pointers instead?

Reply:
I understand that we should not cast unless necessary, the type does not matter.